### PR TITLE
Development vdc S3 expose

### PR DESCRIPTION
**Pull request by waleedhammam**
### Description

- move s3 exposed link to the end of chatflow
- add button for the s3 url in the vdc dashboard

### Related Issues

- https://github.com/threefoldtech/vdc/issues/79

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings

### CODE REVIEW

**Comment by waleedhammam** [comment]
Screenshot from 2020-12-14 14-27-18
Screenshot from 2020-12-14 14-30-12

**Review by AhmedSa-mir — APPROVED** [review]

**Comment by waleedhammam** [comment]
Adding the option to expose if not exposed

**Comment by waleedhammam** [comment]
- Add option to expose after deployment

Screenshot from 2020-12-14 15-59-13

Screenshot from 2020-12-14 15-59-47

Screenshot from 2020-12-14 16-00-05

File: jumpscale/packages/vdc_dashboard/bottle/deployments.py [hunk 1]
```diff
@@ -18,6 +18,24 @@
 app = Bottle()
 
 
+@app.route("/api/s3/expose")
+@login_required
```
Comment (m-motawea) [inline jumpscale/packages/vdc_dashboard/bottle/deployments.py:22]: shouldn't it be is_package_authorized?

**Review by m-motawea — CHANGES_REQUESTED** [review]

File: jumpscale/packages/vdc_dashboard/bottle/deployments.py (hunk 1 above)
Comment (waleedhammam) [reply jumpscale/packages/vdc_dashboard/bottle/deployments.py:22]: done

**Review by m-motawea — APPROVED** [review]

**Pull request merged**